api/src/certification/enrolment/application/certification-candidate-route.js

```diff
@@ -63,6 +63,62 @@ const register = async function (server) {
         ],
       },
     },
+    {
+      method: 'POST',
+      path: '/api/sessions/{id}/certification-candidate',
```

Comment:
Proposition concernant le nom de la nouvelle route.

Comme que nous ne modifions plus la collection de `certification-candidates` mais plutôt la `certification-subscriptions`, on peut penser à renommer cette route pour coller à notre usecase.
À challenger si on rajoute la notion de `certification` dans la route ou non.

```suggestion
path: '/api/sessions/{id}/certification-subscriptions',
```

Reply:
Cette route permet d'ajouter un candidat à la session.
Les souscriptions sont incluses dans l'ajout du candidat mais non exclusives. On ajoute tout un tas d'autres choses.